content/code-security/code-scanning/managing-code-scanning-alerts/managing-code-scanning-alerts-for-your-repository.md

```diff
@@ -200,6 +200,10 @@ To dismiss {% ifversion delete-code-scanning-alerts %}or delete{% endif %} alert
 
 If a project has multiple alerts that you want to dismiss for the same reason, you can bulk dismiss them from the summary of alerts. Typically, you'll want to filter the list and then dismiss all of the matching alerts. For example, you might want to dismiss all of the current alerts in the project that have been tagged for a particular Common Weakness Enumeration (CWE) vulnerability.
 
+## Re-opening dismissed alerts
+
+If you dismiss an alert but later realized that it was done by mistake, you can access the list of closed alerts. Afterwards, you can find the alert, display it and reopen it if necessary.
```

Comment:
Expanding this a little for clarity and so that it aligns better with our style guide.

```suggestion
If you dismiss an alert but later realize that you need to fix the alert, you can re-open it and fix the problem with the code. Display the list of closed alerts, find the alert, display it, and reopen it. You can then fix the alert in the same way as any other alert.
```